superset-frontend/src/SqlLab/components/ColumnElement/index.tsx

```diff
@@ -105,7 +105,7 @@ const ColumnElement = ({ column }: ColumnElementProps) => {
   }
   return (
     <div className="clearfix table-column">
-      <div className="pull-left m-l-10 col-name">
+      <div className="pull-left col-name">
```

Comment:
Do we know where these classes are coming from? Can we use the standard Ant Design approach here, like `Flex` or `Space`?